client/landing/stepper/declarative-flow/internals/steps-repository/store-profiler/index.tsx

```diff
@@ -10,22 +10,26 @@ import FormLabel from 'calypso/components/forms/form-label';
 import FormSelect from 'calypso/components/forms/form-select';
 import FormInput from 'calypso/components/forms/form-text-input';
 import useSiteVerticalsFeatured from 'calypso/data/site-verticals/use-site-verticals-featured';
-import { useCountries } from 'calypso/landing/stepper/hooks/use-countries';
+import { useCountriesAndStates } from 'calypso/jetpack-cloud/sections/partner-portal/company-details-form/hooks/use-countries-and-states';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
-import { SITE_STORE, USER_STORE } from '../../../../stores';
+import { USER_STORE, ONBOARD_STORE } from '../../../../stores';
```

Comment:
Tiny nit: we could switch the import order so they stay alphabetical.
```suggestion
import { ONBOARD_STORE, USER_STORE } from '../../../../stores';
```